packages/data-provider/src/repository/submittedRepository.ts

```diff
@@ -62,11 +62,12 @@ const repository = (dependencies: BaseDependencies) => {
 		/**
 		 * Find SubmittedData by category ID with pagination
 		 * @param {number} categoryId Category ID
+		 * @param {PaginationOps} paginationOps Pagination properties
 		 * @returns The SubmittedData found
 		 */
 		getSubmittedDataByCategoryIdPaginated: async (
 			categoryId: number,
-			paginationOps: paginationOps,
+			paginationOps: PaginationOps,
 		): Promise<SubmittedDataRepository[] | undefined> => {
```

Comment:
1. The Controller is appending to the response [in here](https://github.com/overture-stack/lyric/pull/40/files#diff-a57cfe3761e475ef113627066b842e1c07bed84d8c3569183447f7c06a68e528R172) a summary pagination object containing following fields
```
{
	currentPage: number;
	pageSize: number;
	totalPages: number;
	totalRecords: number;
}
```
2. While querying the database it's possible to get `undefined` when there is no records found. However, on Service Layer file there's already a validation in case no records are found on database an error is throw. [see file](https://github.com/overture-stack/lyric/pull/40/files#diff-29f0e5a306a45d631160b0aea0095a665d9b0ad639ab82a8b8624acff3d6f99fR37)